src/NUnitFramework/framework/Internal/Builders/DefaultTestCaseBuilder.cs

```diff
@@ -141,7 +145,7 @@ public Test BuildFrom(IMethodInfo method, Test parentSuite)
                     tests.Add(test);
             }
 
-            return tests.Count > 0
+            return shouldParameterizeForFewerThanTwoTests || tests.Count > 0
```

Comment:
Okay. I figured I'd make it describe the decision being made, but I'm okay with that.